APIs_Drivers/LowPowerTimer-example/main.cpp

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
could you rename the folder to the example?
something like `lowpowerTimer_ex1` ?